### DISCUSSION TITLE
Branch for Wiki of forked repo invisible in GitHub; Pull Request not possible

### DISCUSSION THREAD

**Discussion by mcalligator** [community/community#38796] [Repositories]
### Select Topic Area

Bug

#### Description
Yesterday, as described in [this related issue](https://github.com/orgs/community/discussions/38790), I forked a repo whose Wiki contained errors (not the code itself), in order to correct them.  Having worked around that issue, I replaced the content with the corrected version in a new branch in my local clone (of the Wiki, rather than the code) and pushed the associated commit to my fork.  I then discovered two connected issues:
1. The new branch is not displayed in GitHub (even though entering `git branch -av` reveals that it does actually exist); and
2. It's therefore not possible to submit a Pull Request to the upstream repo (since I don't have the necessary permissions to merge changes directly to it).

I conducted extensive searches, and confirmed the behaviour with a knowledgeable and experienced colleague, and am now pretty certain it's a bug, rather than intentional.

#### Consistency
Permanent - always occurs

#### Steps to reproduce
1. Navigate to a repo to which one has only read access.
2. Select the Wiki tab of the repo.
3. Fork the repo (clearing the option to **Copy the `master` branch only**).
4. Select the Wiki tab of the forked repo.
5. Clone the Wiki repo to local machine (ensuring the URL has the `.wiki.git` suffix).
5. Create a new branch on the local machine.
6. Create a new page in the Wiki (required due to [this related issue](https://github.com/orgs/community/discussions/38790)).
7. Update that page's content as appropriate.
8. Stage and Commit the change to the local clone.
9. Push change to one's forked repo.
10. Confirm the branch exists both locally and remotely using `git branch -av`.
11. Check the forked repo on GitHub's web UI and confirm that new branch is not visible there.

It is consequently not possible to submit a PR into the upstream repo.

#### Blocking?
No, workaround is to recreate the Wiki content manually

#### Last working date
N/A

#### Logs
N/A

**Comment by jfrantzius** [comment]
I cannot get my fork to contain the Wiki, even though the original repo contains Wiki pages, and I unchecked the "Copy the 'main' branch only" option. I tried adding a page to the empty Wiki as suggested in the related issue, and then cloned it, but it's still empty except for my newly created page.

**Comment by chemelli74** [comment]
Confirmed to be stil an issue
